features/cerberusweb.core/api/dao/worker_role.php

```diff
@@ -340,7 +340,7 @@ static function getCumulativePrivsByWorker($worker_id, $nocache=false) {
 					case 'all':
 						$privs = ['*' => []];
 						$cache->save($privs, self::_CACHE_WORKER_PRIVS_PREFIX.$worker_id);
-						return;
+						return $privs;
```

Comment:
I'll change this to:

```suggestion
						break 2;
```

That way we still exit the outer loop, cache the result, and return `$privs`.